Prob3/Program.cs

```diff
@@ -0,0 +1,159 @@
+﻿/**
+ * Definition for singly-linked list.
+ * public class ListNode {
+ *     public int val;
+ *     public ListNode next;
+ *     public ListNode(int val=0, ListNode next=null) {
+ *         this.val = val;
+ *         this.next = next;
+ *     }
+ * }
+ */
+public class Solution
+{
```

Comment:
@noor848 Where is the` Main` method?? 
and where is the` ListNode` class?? (remove the comment above)

Reply:
fixed